GenderPayGap.WebUI/Controllers/AddOrganisation/AddOrganisationManualController.cs

```diff
@@ -1,33 +0,0 @@
-﻿using GenderPayGap.Core;

```

Comment:
This file has been split into `AddOrganisationManualNameController` and `AddOrganisationManualAddressController`